steps/xcodegen/0.0.2/step.yml

```diff
@@ -0,0 +1,66 @@
+title: XcodeGen
+summary: |
+  This Bitrise step runs XcodeGen to generate a project.
+description: |
+  This Bitrise step runs XcodeGen to generate a project.
+website: https://github.com/brandtdaniels/xcodegen-bitrise
+source_code_url: https://github.com/brandtdaniels/xcodegen-bitrise
+support_url: https://github.com/brandtdaniels/xcodegen-bitrise/issues
+published_at: 2019-10-15T15:34:57.183896-07:00
+source:
+  git: https://github.com/brandtdaniels/xcodegen-bitrise.git
+  commit: 3457781eb072e5673ddfa3a35806edc21eda8236
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- macos
```

Comment:
Works for tvos and watchos as well. Don’t know if they are specific types in bitrise

Reply:
These are optional and only used for filtering

> project_type_tags : project type tags if the Step is project type specific. Example: ios or android. Completely optional, and only used for search and filtering in Step lists.